templates/info.hpp.in

```diff
@@ -48,6 +48,7 @@ namespace @ModelName@_info {
    constexpr bool is_low_energy_model = @isLowEnergyModel@;
    constexpr bool is_supersymmetric_model = @isSupersymmetricModel@;
    constexpr bool is_FlexibleEFTHiggs = @isFlexibleEFTHiggs@;
+   constexpr bool CPViolationInHiggsSector {@CPViolationInHiggsSector@};
```

Comment:
can we rename it to something that starts with `is_`? Like for example `is_CP_violating_Higgs_sector`? This would fit a bit better to the other bools.

Reply:
Done. Thanks for the comment. Anything else?